composer/algorithms/augmix/augmix.py

```diff
@@ -83,29 +118,49 @@ def forward(self, img: ImageType) -> ImageType:
+    """AugMix (`Hendrycks et al., 2020 <http://arxiv.org/abs/1912.02781>`_) creates
+    ``width`` sequences of ``depth`` image augmentations, applies each sequence with
+    random intensity, and returns a convex combination of the ``width`` augmented images
+    and the original image.  The coefficients for mixing the augmented images are drawn
+    from a uniform ``Dirichlet(alpha, alpha, ...)`` distribution. The coefficient for
+    mixing the combined augmented image and the original image is drawn from a
     ``Beta(alpha, alpha)`` distribution, using the same ``alpha``.
 
     This algorithm runs on on :attr:`Event.FIT_START` to insert a dataset transformation. It is a no-op if this algorithm already
     applied itself on the :attr:`State.train_dataloader.dataset`.
 
+    Example:
+        .. testcode::
+
+            from composer.algorithms import AugMix
+            from composer.trainer import Trainer
+            augmix_algorithm = AugMix(
+                severity=3,
+                width=3,
+                depth=-1,
+                alpha=1.0,
+                augmentation_set="all"
+            )
+            trainer = Trainer(
+                model=model,
+                train_dataloader=train_dataloader,
+                eval_dataloader=eval_dataloader,
+                max_duration="1ep",
+                algorithms=[augmix_algorithm],
+                optimizers=[optimizer]
+            )
+
     Args:
-        severity: severity of augmentations; ranges from 0
-            (no augmentation) to 10 (most severe).
-        width: number of augmentation sequences
-        depth: number of augmentations per sequence. -1 enables stochastic depth
-            sampled uniformly from [1, 3].
-        alpha: pseudocount for Beta and Dirichlet distributions. Must be > 0.
-            Higher values yield mixing coefficients closer to uniform
-            weighting. As the value approaches 0, the mixing coefficients
-            approach using only one version of each image.
-        augmentation_set: must be one of the following options:
+        severity (int, optional): severity of augmentations; ranges from 0
+            (no augmentation) to 10 (most severe). Default = ``3``.
+        width (int, optional): number of augmentation sequences. Default = ``3``.
+        depth (int, optional): number of augmentations per sequence. -1 enables stochastic
+            depth sampled uniformly from [1, 3]. Default = ``-1``.
+        alpha (float, optional): pseudocount for Beta and Dirichlet distributions. Must be
+            > 0.  Higher values yield mixing coefficients closer to uniform weighting. As
+            the value approaches 0, the mixing coefficients approach using only one
+            version of each image. Default = ``1.0``.
+        augmentation_set (str, optional): must be one of the following options:
 
             * ``"augmentations_all"``
                 Uses all augmentations from the paper.
```

Comment:
The description for `augmentations_original` could use some formatting for the equations.